onert-micro/luci-interpreter/src/kernels/BroadcastTo.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Builders.h"
+#include "kernels/Utils.h"
+#include "TISOKernel.h"
+
+#include "PALBroadcastTo.h"
+
+namespace luci_interpreter
+{
+namespace
+{
+constexpr int kMaxDims = 5;
```

Comment:
@BalyshevArtem Hello. During working on `https://github.com/Samsung/ONE/pull/11701#issuecomment-1774450755`, I met a out-bound error on this. Seems that tensorflow supports up to 8 kMaxDims but 8 occurs an error. Could you let me know why you set `kMaxDims` 5?

https://github.com/tensorflow/tensorflow/blob/1b8f5c396f0c016ebe81fe1af029e6f205c926a4/tensorflow/lite/kernels/broadcast_to.cc#L118